package.json

```diff
@@ -90,17 +90,17 @@
     "react-file-drop": "^0.1.8",
     "react-map-gl": "^3.2.1",
     "react-modal": "^3.1.10",
-    "react-onclickoutside": "https://registry.npmjs.org/react-onclickoutside/-/react-onclickoutside-6.7.1.tgz",
```

Comment:
the url is specified because if using unpm registry, this will try to get @uber/react-onclickoutside, although I haven't check, is that still the case?

Reply:
It's no longer the case. We also ensure that it's not the case through the yarn.lock .